web/packages/teleport/src/features.tsx

```diff
@@ -100,12 +100,13 @@ class AccessRequests implements TeleportFeature {
 
   navigationItem = {
     title: NavTitle.AccessRequests,
-    icon: <EqualizersVertical />,
+    icon: <EqualizersVertical color="text.main" />,
     exact: true,
     getLink() {
       return cfg.routes.accessRequest;
     },
   };
+  topMenuItem = this.navigationItem;
```

Comment:
Instead of this being a copy of `navigationItem`, can it be an optional boolean?

Reply:
the type of `topMenuItem` is used elsewhere as a component that shares a lot of the same props as `navigationItem`. I could make the type be `TopMenuItem | boolean` but that might muck it up instead of just copying over like this